from shaas_bot.handlers.system.error import handle_bot_exception

system_handlers = [
]

error_handlers = [
    handle_bot_exception
]